core/src/integration-test/java/com/scalar/db/storage/jdbc/test/TestEnv.java

```diff
@@ -1,107 +1,33 @@
 package com.scalar.db.storage.jdbc.test;
 
-import com.google.common.collect.ImmutableMap;
-import com.scalar.db.api.Scan;
+import static com.scalar.db.util.Utility.getFullNamespaceName;
+
 import com.scalar.db.api.TableMetadata;
 import com.scalar.db.config.DatabaseConfig;
-import com.scalar.db.io.DataType;
+import com.scalar.db.exception.storage.ExecutionException;
 import com.scalar.db.storage.jdbc.JdbcConfig;
+import com.scalar.db.storage.jdbc.JdbcDatabaseAdmin;
 import com.scalar.db.storage.jdbc.JdbcTableMetadataManager;
 import com.scalar.db.storage.jdbc.JdbcUtils;
 import com.scalar.db.storage.jdbc.RdbEngine;
 import com.scalar.db.storage.jdbc.query.QueryUtils;
+import com.scalar.db.util.Utility;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import java.io.Closeable;
 import java.io.IOException;
 import java.sql.Connection;
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.ArrayList;
-import java.util.HashSet;
+import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.Optional;
 import java.util.Properties;
-import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-import javax.sql.DataSource;
 import org.apache.commons.dbcp2.BasicDataSource;
 
 @SuppressFBWarnings("SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE")
 public class TestEnv implements Closeable {
```

Comment:
A lof of changes in the PR were due to the refactoring of this class to use the `JdbcDatabaseAdmin` to prepare a integration test environment.